test/js/bun/shell/bunshell.test.ts

```diff
@@ -148,8 +153,9 @@ describe("bunshell", () => {
 
     test("escape unicode", async () => {
       const { stdout } = await $`echo \\弟\\気`;
-
-      expect(stdout.toString("utf8")).toEqual(`\弟\気\n`);
+      // expect(stdout.toString("utf8")).toEqual(`\弟\気\n`);
```

Comment:
```suggestion
      // TODO: Uncomment and replace after unicode in template tags is supported
      // expect(stdout.toString("utf8")).toEqual(`\弟\気\n`);
```
Adding the TODO comment helps for folks cleaning house (or doing tech debt work)